Automattic/wp-calypso
# Stats: Make Upsell section recognize Complete plans

Related to:

- Issue: https://github.com/Automattic/wp-calypso/issues/80536
- Project: pejTkB-1zl-p2

## Proposed Changes

Odyssey Stats shows an Upsell section at the bottom of the page. With this change, sites with a Complete plan will no longer see the upsell.

## Why are these changes being made?


The upsell doesn't make sense for sites with a Complete plan.

## Testing Instructions



* Set up per FG instructions: PCYsg-Pp7-p2
* Check a site without a Complete plan and confirm the upsell is present.
* Check a site with a Complete plan and confirm the upsell is no longer shown.

Note: The request is cached on the server side (I think) so the `purchases` API won't immediately reflect plan changes made through Store Admin. Better to test on two separate sites. You can confirm the page is properly reflecting the API response by inspecting it in the dev tools.

## Pre-merge Checklist
